package.json

```diff
@@ -65,6 +65,10 @@
     "http-server": "^14.1.0",
     "lint-staged": "~11.2.6",
     "postcss": "~8.3.11",
+    "postcss-dir-pseudo-class": "^6.0.4",
```

Comment:
We should use install `postcss-dir-pseudo-class` with the `~6.0.4` patch releases modifier like the other ones, right?

Reply:
fixed